test/resources/kubeconfig.resource

```diff
@@ -0,0 +1,50 @@
+*** Settings ***
+Library    OperatingSystem
+Library    SSHLibrary
+Library    String
+
+*** Keywords ***
+Get Kubeconfig
+    [Documentation]    Get the kubeconfig file from the host argument and return contents
+    [Arguments]    ${host}
+    SSHLibrary.Open Connection    ${host}
+    SSHLibrary.Login              ${USHIFT_USER}    allow_agent=True
+    ${kubeconfig}    ${rc}=    Execute Command
+    ...    cat /var/lib/microshift/resources/kubeadmin/${host}/kubeconfig
+    ...    sudo=True    return_rc=True
+    SSHLibrary.Close Connection
+    Should Be Equal As Integers    ${rc}    0
+    Should Not Be Empty            ${kubeconfig}
+    [Return]    ${kubeconfig}
+
+Setup Kubeconfig
+    [Documentation]    Get the kubeconfig file from the configured $USHIFT_HOST, create a temporary file
+    ...                and export it as $KUBECONFIG variable.
+    ${kubeconfig}=    Get Kubeconfig    ${USHIFT_HOST}
+    ${rand}=    Generate Random String
+    ${path}=    Join Path    /tmp    ${rand}
+    Create File    ${path}    ${kubeconfig}
+    Set Suite Variable    \${KUBECONFIG}    ${path}
+
+Remove Kubeconfig
+    [Documentation]    Remove kubeconfig ${KUBECONFIG} file.
+    Remove File    ${KUBECONFIG}
```

Comment:
I'm not clear on how environment variables work in RF. Is there any danger of my real kubeconfig being deleted if I have the KUBECONFIG environment variable set?